### ISSUE TITLE
Android netplay?

### ISSUE THREAD

**Issue by 83RL** [zach-morris/plugin.program.iarl#61]
![capture](https://user-images.githubusercontent.com/32914715/34323344-3a99985e-e813-11e7-991e-011c204e1cc7.PNG)

it doesnt work for android or hasnt been added yet?   i set to host,  changed name and launched game and no "Waiting for client" upon launch.

i also externally checked lobby just in case but no luck.

**Comment by zach-morris** [comment]
Yes, unfortunately, Android does not support command line netplay options.  Thats a limitation of retroarch currently, rather than IARL.
